web-app/HeimScripts/heatmap/run.R

```diff
@@ -73,7 +74,7 @@ main <- function(max_rows = 100, sorting = "nodes", ranking = "coef") {
 writeMarkerTable <- function(markerTable){
   colnames(markerTable) <- c("rowLabel", "biomarker",
                              "log2FoldChange", "t", "pValue", "adjustedPValue", "B")
-  jsn                   <- toJSON(markerTable, pretty = TRUE)
+  jsn                   <- toJSON(markerTable, pretty = TRUE, digits = 17)
   write(jsn, file = markerTableJson)
 }
 
```

Comment:
Around line 300 there is also a   write(toJSON(params, pretty = TRUE), 'params.json')
in params: are there any numeric values that might be influenced too? in fetch_params for example?